BaikalIO/material_io.cpp

```diff
@@ -227,13 +227,13 @@ namespace Baikal
         UberV2Material::Ptr material = UberV2Material::Create();
         material->SetLayers(std::atoi(element.Attribute("layers")));
         material->SetThin(thin == "true");
-        material->SetDoubleSided(element.Attribute("emission_doublesided") == "true");
-        material->LinkRefractionIOR(element.Attribute("refraction_link_ior") == "true");
-        material->SetMultiscatter(element.Attribute("sss_multyscatter") == "true");
+        material->SetDoubleSided(strcmp(element.Attribute("emission_doublesided"), "true") == 0);
+        material->LinkRefractionIOR(strcmp(element.Attribute("refraction_link_ior"), "true") == 0);
+        material->SetMultiscatter(strcmp(element.Attribute("sss_multyscatter"), "true") == 0);
         material->SetName(name);
 
         auto num_inputs = material->GetNumInputs();
-        for (size_t a = 0; a < num_inputs; ++a)
+        for (uint32_t a = 0u; a < num_inputs; ++a)
```

Comment:
Material::GetInput requires std::uint32_t. Maybe it would be better to change its parameter type to size_t? It works with std::vector and vector::value_type is size_t.